db/migrate/20210203095633_add_webauthn_id_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddWebauthnIdToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :webauthn_id, :string
```

Comment:
Is this set up so a user can only have one WebAuthn credential?

If we were doing this for real we'd want to support multiple credentials per account, as the credentials (can be) device specific.

Reply:
Yeah so this does support multiple credentails.
I named this `webauthn_id` because that's what the ruby library has it as. I think we could rename to clarify though.

This is more like the `user_webauthn_id`, it identfies a user not a credential.

Reply:
Ah, I see, this is a ["user handle"](https://www.w3.org/TR/webauthn-2/#user-handle) - a user ID in webauthn-speak.

Reply:
yeah that'd be a much better name for it if I was doing this again